src/main/java/com/textmessenger/controller/UserController.java

```diff
@@ -38,6 +38,13 @@ public UserController(UserService userService) {
             .orElse(ResponseEntity.notFound().build());
   }
 
+  @PostMapping("/find")
+  public ResponseEntity<?> findAllUsers(@RequestBody String str) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove usage of generic wildcard type. ([squid:S1452](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1452))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=121&issues=AWU4Qf8vW2AOP0CkHewC&open=AWU4Qf8vW2AOP0CkHewC)

[ISSUE_KEY]: AWU4Qf8vW2AOP0CkHewC